src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -229,23 +230,91 @@ def load_command_table(self, _):
         g.custom_command('export', 'export_deployment_as_template')
         g.wait_command('wait')
 
-    with self.command_group('group deployment operation', resource_deployment_operation_sdk) as g:
+    with self.command_group('group deployment operation', resource_deployment_operation_sdk, deprecate_info=self.deprecate(redirect='deployment operation group', hide=True)) as g:
         g.command('list', 'list')
         g.custom_show_command('show', 'get_deployment_operations', client_factory=cf_deployment_operations)
 
+    # az deployment
     with self.command_group('deployment', resource_deployment_sdk, min_api='2018-05-01', resource_type=ResourceType.MGMT_RESOURCE_RESOURCES) as g:
```

Comment:
You also need to deprecate this command group because it is changed to `deployment sub`, right?